builtin/aws/lambda/platform.go

```diff
@@ -933,6 +965,12 @@ type Config struct {
 	// Must be a value between 512 and 10240.
 	// Defaults to 512 MB.
 	StorageMB int `hcl:"storagemb,optional"`
+
+	// Environment variables that are meant to configure the application in a static
+	// way. This might be control an image that has multiple modes of operation,
+	// selected via environment variable. Most configuration should use the waypoint
+	// config commands.
```

Comment:
I love this doc comment! We should include it in the `doc.SetField` too 😄

Reply:
Added a summary!